TwilioVerifySDK/TwilioVerify/Sources/API/FactorAPIClient.swift

```diff
@@ -21,7 +21,7 @@ import Foundation
 
 protocol FactorAPIClientProtocol {
   func create(withPayload payload: CreateFactorPayload, success: @escaping SuccessResponseBlock, failure: @escaping FailureBlock)
-  func verify(_ factor: Factor, authPayload: String, success: @escaping SuccessResponseBlock, failure: @escaping FailureBlock)
+  func verify(_ factor: Factor, allowIphoneMigration: Bool, authPayload: String, success: @escaping SuccessResponseBlock, failure: @escaping FailureBlock)
```

Comment:
Could the verify method get the `allowIphoneMigration` value from the `factor`? So, we will use the defined option (allow or not) when creating the factor instead of requesting it again, so we can prevent errors (creating with allow=true, but calling verify with false)